config/routes.rb

```diff
@@ -5,8 +5,10 @@
 	mount Sidekiq::Web => '/sidekiq'
 	
 	mount_devise_token_auth_for 'User', at: 'auth'
-	resources :books, only: [:index, :show]
+	resources :books, only: [:index, :show, :open_library_book]
```

Comment:
You should not add `open_library_book` to the _only_ route, _only_ routes should be reserve for CRUD operation. 
A good aproach would be to add a resources for the open library service, like:
```
resources :books, only: [:index, :show] do
  collection do
    get :open_library_nook
  end
end
```

Or maybe anothe better aproach would be to move it to another controller where you could add the CRUD interface, which in my opinion is a better aproach:

```
resouces :open_library, only: [:show]
```

For more info about routes you could check => https://guides.rubyonrails.org/routing.html